core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -97,7 +97,10 @@ <h4 translate="I18N_PLAYER_SHARE_THIS_EXPLORATION" class="share-exploration-foot
   }
 
   .oppia-exploration-footer .dropdown-menu.author-profile-dropdown-menu {
-    border-radius: 0;
+    border: none !important;
+    border-bottom-left-radius: 0px;
+    border-bottom-right-radius: 0px;
+    box-shadow: 0px -6px 12px rgba(0, 0, 0, 0.176);
```

Comment:
I can go ahead and make these changes, but if you look at the rest of the file, you'd see 0px being used in other places. 

Should I get rid of those too?